snuba/manual_jobs/__init__.py

```diff
@@ -0,0 +1,22 @@
+from abc import ABC, abstractmethod
+
+from snuba.utils.registered_class import RegisteredClass
+
+
+class Job(ABC, metaclass=RegisteredClass):
+    def __init__(self, dry_run: bool, **kwargs) -> None:
+        self.dry_run = dry_run
+        for k, v in kwargs.items():
+            setattr(self, k, v)
+
+    @abstractmethod
+    def _build_query(self) -> str:
+        pass
```

Comment:
I don't think this should be a method on the abstract class (I can imagine some jobs may not result in ClickHouse queries)